addons/aws-efs-csi-driver/0.3.x/efs-1.yaml

```
@@ -0,0 +1,23 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: aws-efs-csi-driver
+  labels:
+    kubeaddons.mesosphere.io/name: aws-efs-csi-driver
+    kubeaddons.mesosphere.io/provides: aws-efs-csi-driver
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.3.0-1"
+    appversion.kubeaddons.mesosphere.io/aws-efs-csi-drive: "0.3.0"
+    values.chart.helm.kubeaddons.mesosphere.io/aws-efs-csi-drive: "https://raw.githubusercontent.com/kubernetes-sigs/aws-efs-csi-driver/master/helm/values.yaml"
```

```suggestion
    values.chart.helm.kubeaddons.mesosphere.io/aws-efs-csi-drive: "https://raw.githubusercontent.com/kubernetes-sigs/aws-efs-csi-driver/6992149/helm/values.yaml"
```
we should point here to the releases commit value